articles/dashboard/guides/tenants/view-application-signing-keys.md

```diff
@@ -0,0 +1,93 @@
+---
+title: View Application Signing Keys
+description: Learn how to view your tenant's application signing keys, which are used to sign tokens and SAML assertions, using the Auth0 Dashboard and Auth0 Management API.
+topics:
+  - tokens
+  - access-tokens
+  - id-tokens
+  - assertions
+  - SAML
+  - signing-keys
+  - certificates
+  - dashboard
+  - mgmt-api
+contentType:
+  - how-to
+useCase:
+  - secure-api
+  - add-login
+  - call-api
+---
+
+# View Application Signing Keys
```

Comment:
Maybe on this page we can also include the language about how the signing keys for connections are different, and how to get it (/pem?cert=connection e.g.)